spec/base.spec.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+define(['ably', 'globals'], function(Ably, ablyGlobals) {
+  describe('Static generated ably.js library', function() {
+    it('is loaded into the global namespace', function() {
+      expect(Ably).not.toBeUndefined();
+    });
+  });
+
+  describe('Globals in support library', function() {
+    var isBrowser = (typeof(window) == 'object');
+    var environment = isBrowser ? window.__env__ : process.env;
+
+    it('contains a valid environment', function() {
+      expect(ablyGlobals.environment).toEqual(environment.ABLY_ENV || 'sandbox');
```

Comment:
Well no, because then if you change the environment of your test i.e. ABLY_ENV=staging, the test will fail.  This is an example test anyway and will be removed, it was used to test Karma / Jasmine to make sure the globals were accessible.